### PR TITLE
Fixed typo in org description

### PR DESCRIPTION
**Pull request by rlenferink**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/internal-acls/78)

### CODE REVIEW

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ orgs:
         company: ""
         default_repository_permission: read
         description: Kubeflow is an open, community driven project to make it easy to deploy
-          and manage an ML stack on Kubernetes
+          and manage a ML stack on Kubernetes
```
Comment (jlewi) [inline github-orgs/kubeflow/org.yaml]: @sarahmaddox Can you just confirm it should be "a" not "an"

Comment (sarahmaddox) [reply github-orgs/kubeflow/org.yaml]: It should be "an" because you'd pronounce the phrase as "an ML stack" not "a machine learning stack". :)

Comment (rlenferink) [reply github-orgs/kubeflow/org.yaml]: Well, it sounded at bit strange to me, so I thought it should be 'a machine learning stack'. But it seems I've learned another thing for today 😄 

Just for my own curiosity (non-native English speaker), how is this different from `a machine learning algorithm` (used [here](https://www.dictionary.com/browse/machine-learning) and [here](https://en.wikipedia.org/wiki/Machine_learning#Applications)) or `a machine learning system` (used [here](https://www.linkedin.com/pulse/pros-cons-two-approaches-machine-learning-grammar-engineering-wei-li/)).

Or from the verb `a machine` [here](https://www.macmillandictionary.com/dictionary/british/machine_2)

Comment (jlewi) [reply github-orgs/kubeflow/org.yaml]: I've gone ahead and closed the PR. Feel free to continue the discussion.

Grammar was never my strong point so I don't have any insight on why one is more correct.

**Pull request closed without merging**